README.md

```diff
@@ -53,6 +53,101 @@ Add this to your project's `pom.xml` file:
 ```
 
 
+## Getting Started
+
+### Add your Android Platform
+To initialize your SDK and start interacting with Appwrite services, you need to add a new Android platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.
+
+From the options, choose to add a new **Android** platform and add your app credentials.
+
+Add your app <u>name</u> and <u>package name</u>. Your package name is generally the applicationId in your app-level `build.gradle` file. By registering a new platform, you are allowing your app to communicate with the Appwrite API.
+
+### OAuth
+In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your [AndroidManifest.xml](). Be sure to replace the **[PROJECT_ID]** string with your actual Appwrite project ID. You can find your Appwrite project ID in your project settings screen in the console.
```

Comment:
Missing a link for an example `AndroidManifest.xml` file - we can use the one in our playground

Reply:
They one in the playground doesn't have the callback activity yet. Probably once Damodar updates it, I can add it. 
What do you say ? @eldadfux

Reply:
👍 sure, but let's set a reminder.